modules/assim.sequential/R/SDA_OBS_Assembler.R

```diff
@@ -0,0 +1,175 @@
+#' Assembler for preparing obs.mean and obs.cov for the SDA workflow
+#'
+#' @param settings_dir the path of settings.xml object.
+#' @param Var Variable name, currently support: SMP, AGB, and LAI.
+#' @param OutDir the path to store obs.mean and obs.cov
+#' @param Obs_Prep if your settings object doesn't contain Obs_Prep, you can import it separately (details see L17-18).
+#' @param skip_buffer flag to skip calculating min var based on buffer area for agb data.
+#'
+#' @return list of obs.mean and obs.cov
+#' @export
+#' @author Dongchen Zhang
+#'
+#' @examples
+#' \dontrun{
+#' settings_dir <- "/projectnb/dietzelab/dongchen/All_NEON_SDA/NEON42/SDA/pecan.xml"
+#' OutDir <- "/projectnb/dietzelab/dongchen/All_NEON_SDA/test_OBS"
+#' Var <- c("SMP", "LAI", "AGB")
+#' OBS <- SDA_OBS_Assembler(settings_dir, Var, OutDir)
+#' }
+
+
+SDA_OBS_Assembler <- function(settings_dir, Var, OutDir, Obs_Prep = NULL, skip_buffer = TRUE){
+  #export special operator
+  `%>%` <- magrittr::`%>%` 
```

Comment:
These should be handled using 
```#' @importFrom magrittr %>%```
in the roxygen. See for example assim.sequential/R/Helper.functions.R

Reply:
fixed